src/utils/languages.ts

```diff
@@ -17,6 +17,27 @@ export const availableLanguages: Language[] = [
   { code: 'ja', name: '日本語', flag: '🇯🇵' },
   { code: 'ru', name: 'Русский', flag: '🇷🇺' },
   { code: 'th', name: 'ไทย', flag: '🇹🇭' },
+  { code: 'en', name: 'English', flag: '🇺🇸' },
+  { code: 'ko', name: '한국어', flag: '🇰🇷' },
+  { code: 'pt', name: 'Português', flag: '🇵🇹' },
+  { code: 'ar', name: 'العربية', flag: '🇸🇦' },
```

Comment:
In order to support Arabic, I think we need to support right-justification as well as text.
https://en.wikipedia.org/wiki/Right-to-left_script

So, it would be better to remove it once.

Reply:
Yes I'm thinking about the RTL integration...